doc/source/HOW-TO/cluster-deployment.rst

```diff
@@ -0,0 +1,74 @@
+********************************************************
+Build and install a custom version of Qserv on a cluster
+********************************************************
+
+Build on a dedicated machine
+============================
+
+Pre-requisites
+++++++++++++++
+
+* Install LSST stack and Qserv dependencies as described here :ref:`quick-start-devel-pre-requisites`.
+* Open write access for each developer to Qserv install path:
+
+  .. code-block:: bash
+
+   # Update acl
+   # TODO: test it
```

Comment:
I need somebody else with a account on the cluster to confirm me it works. I removed the comment.